tools/WixRelease.relproj

```diff
@@ -61,6 +61,12 @@
         <Title>WiX v$(MajorMinorVersionDottedString) source and symbols for debugging purposes.</Title>
         <Protected>true</Protected>
       </ReleaseFile>
+      
+      <!-- Upload the Nuget package to the website, at least until it gets published to nuget.org -->
+      <ReleaseFile Include="$(OutputPath)WixToolset.Tools.$(FullBuildVersionString).nupkg">
+        <Title>WiX v$(MajorMinorVersionDottedString) WixToolset.Tools Nuget package.</Title>
+        <Protected>true</Protected>
+      </ReleaseFile>
```

Comment:
No, we'll publish to NuGet.org when it is time. No need to put it on our storage unless you see some reason to keep them on our storage as well as NuGet.org?